src/main/java/com/pretz/dyntreeserver/service/DynTreeService.java

```diff
@@ -0,0 +1,26 @@
+package com.pretz.dyntreeserver.service;
+
+import com.pretz.dyntreeserver.domain.DynTree;
+import com.pretz.dyntreeserver.generator.DynTreeGenerator;
+import com.pretz.dyntreeserver.service.dto.DynTreeDTO;
+import com.pretz.dyntreeserver.service.dto.DynTreeMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class DynTreeService {
+
+    private DynTreeMapper dynTreeMapper;
+    private DynTreeGenerator dynTreeGenerator;
```

Comment:
is it necessary to split logic so far down?

Reply:
why generator logic can't be here in this magic `DynTreeService`?

What else it'll do?

Reply:
Ok, basing on my absolutely amazing analysis of data flow here https://trello.com/c/vxywFE9D/10-create-tree-function-on-server this service will also control the saving of dynasty trees confirmed and/or edited by user.